### PR TITLE
Feature/1083 cleanup cortex extra

### PR DESCRIPTION
**Pull request by sgryphon**
Cleanup of things I had in cortex-extra

### CODE REVIEW

File: src/Nethermind/Nethermind.BeaconNode.Host/Nethermind.BeaconNode.Host.csproj [hunk 1]
```diff
@@ -24,5 +24,14 @@
     <ProjectReference Include="..\Nethermind.BeaconNode\Nethermind.BeaconNode.csproj" />
     <ProjectReference Include="..\Nethermind.BeaconNode.Storage\Nethermind.BeaconNode.Storage.csproj" />
   </ItemGroup>
+
+  <ItemGroup>
+    <Content Remove="configuration\**" />
```
Comment (tkstanczak) [inline src/Nethermind/Nethermind.BeaconNode.Host/Nethermind.BeaconNode.Host.csproj]: was it planned?

Comment (sgryphon) [reply src/Nethermind/Nethermind.BeaconNode.Host/Nethermind.BeaconNode.Host.csproj]: Planned to not be included in the output? Yes. 

The configuration folder in that project has the YAML files from the specs., and a powershell script that converts them to JSON snippets, that I then paste into appsettings.json. The appsettings files are used in the published version, not the YAML.

The intention is that the appsettings files are standard for .NET Core and will have examples for third party apps. e.g. Serilog has config examples in JSON. (The actual config system won't care, command line, environment, etc.; it is more for a dev familiar with .NET).

Note that the underlying code also supports the YAML file, if that is wanted. Either will work (or command line, environment, etc).

**Comment by sgryphon** [comment]
> Is it possible to remove the dlls (from nswag tools and gitversion runtime?)
> (lot of heavy dlls landing in the repo)

Sure. I'm not sure how you manage tools in your project, but happy with whatever approach you take. 

I stuck them in my repo so that I know my scripts work with the version of the tool used. Kind of copied from other projects where the build server may not have the tools (but does have .NET) so uses the ones from the repo.

Maybe the idea of downloading the tool as part of the script might work. I'll have a look.

File: src/Nethermind/Nethermind.BeaconNode.Host/README.md [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+
+Client code:
+
+```
+dotnet tools/nswag/dotnet-nswag.dll openapi2csclient /input:oapi/beacon-node-oapi.yaml /classname:BeaconNodeClient /namespace:Nethermind.BeaconNode.ApiClient /ContractsNamespace:Nethermind.BeaconNode.ApiClient.Contracts /output:../Nethermind.BeaconNode.ApiClient/BeaconNodeClient-generated.cs
+```
+
+### Implemented
+
+Impelmented so far:
```
Comment (tkstanczak) [inline src/Nethermind/Nethermind.BeaconNode.Host/README.md]: typo, leaving it for a post merge fix

**Comment by sgryphon** [comment]
> Is it possible to remove the dlls (from nswag tools and gitversion runtime?)
> (lot of heavy dlls landing in the repo)

Done. 

gitversion added as msbuild task -- actually really simple, and even better than my scripted tool, as it applies to all builds, e..g plain "dotnet build" now has a version number.

nswag added as <DotNetCliToolReference>

**Pull request merged**